lumicks/pylake/detail/imaging_mixins.py

```diff
@@ -1,11 +1,103 @@
+import json
+import numpy as np
+import numpy.typing as npt
+import tifffile
+from typing import Iterator, Union
+
 from ..adjustments import ColorAdjustment
 from .timeindex import to_timestamp
-import numpy as np
-from typing import Union
 
 _FIRST_TIMESTAMP = 1388534400
 
 
+class TiffExport:
+    def export_tiff(self, filename, *, dtype=None, clip=False):
```

Comment:
I would expect this function to be called `export`, or `export_tiff_stack` since you are not just exporting a single frame.